api/test/resources/workflows/assemble_refbased/assemble_refbased.wdl

```diff
@@ -0,0 +1 @@
+https://cdn.jsdelivr.net/gh/broadinstitute/viral-pipelines@master/pipes/WDL/workflows/assemble_refbased.wdl
```

Comment:
should we pin to a commit/tag of the `broadinstitute/viral-pipelines` repo?

Reply:
Good idea. I'll fix all these :D